chaos-experiments/docker-compose/scripts/verify-steady-state.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+echo "deploying test workflow…"
+zbctl --insecure deploy ../../bpmn/diagram_1.bpmn
+
+echo "Starting test workflow run for 60s…"
+zbctl --insecure create instance benchmark
```

Comment:
Since you have a message catch event, we could try the following out.

Deploy the workflow, send a message, start an instance and await the result. With this we could make sure that the complete cycle still works. What do you think? To await the result we just need to add the `--withResult` flag.